README.md

```diff
@@ -0,0 +1,34 @@
+# vatvalidate
```

Comment:
I'd maybe add a reference to the algorithm in the readme

Reply:
https://github.com/Beauhurst/vatvalidate/pull/1/commits/77e41d434c8a75641add8e73ab1f4a12c12c290c :+1: